queries/liquid/injections.scm

```diff
@@ -0,0 +1,21 @@
+((template_content) @injection.content
+  (#set! injection.language "html")
+  (#set! injection.combined))
```

Comment:
Please ensure that there is never any two injection queries that capture the same node

Reply:
I went ahead and explicitly defined these node types here separately in the grammar to prevent them from being captured multiple times.